package/cfg/k3s-cis-1.9/policies.yaml

```
@@ -107,7 +107,7 @@ groups:
         scored: true
 
       - id: 5.1.5
-        text: "Ensure that default service accounts are not actively used. (Automated)"
+        text: "Ensure that default service accounts are not actively used. (Manual)"
```

```suggestion
        text: "Ensure that default service accounts are not actively used. (Automated)"
```
I would lean towards keeping this `automated` as we can audit K3s in an automated way, then add whitelisting if needed. This will likely make the check to `FAIL` instead of `WARN`, but then as I you stated, the user can manually fix by following the doc.

---

Removed that change from PR